plasmapy/formulary/parameters.py

```diff
@@ -553,43 +685,47 @@ def thermal_speed(
     T: u.K,
     particle: Particle,
     method="most_probable",
-    mass: u.kg = np.nan * u.kg,
+    mass: u.kg = None,
     ndim=3,
 ) -> u.m / u.s:
     r"""
-    Return the most probable speed for a particle within a Maxwellian
-    distribution.
+    Calculate the speed of thermal motion for particles with a Maxwellian
+    distribution.  (See the :ref:`Notes <thermal-speed-notes>` section for
+    details.)
+
+    **Aliases:** `~plasmapy.formulary.parameters.vth_`
 
-    **Aliases:** `vth_`
+    **Lite Version:** `~plasmapy.formulary.parameters.thermal_speed_lite`
 
     Parameters
     ----------
     T : `~astropy.units.Quantity`
-        The particle temperature in either kelvin or energy per particle
+        The temperature of the particle distribution, in units of kelvin or
+        energy.
 
-    particle : `~plasmapy.particles.Particle`
-        Representation of the particle species (e.g., ``'p'`` for protons, ``'D+'``
-        for deuterium, or ``'He-4 +1'`` for singly ionized helium-4). If no
-        charge state information is provided, then the particles are
+    particle : `~plasmapy.particles.particle_class.Particle`
+        Representation of the particle species (e.g., ``"p"`` for protons,
+        ``"D+"`` for deuterium, or ``"He-4 +1"`` for singly ionized helium-4).
+        If no charge state information is provided, then the particles are
         assumed to be singly charged.
 
     method : `str`, optional
-        Method to be used for calculating the thermal speed. Options are
-        ``'most_probable'`` (default), ``'rms'``, and ``'mean_magnitude'``.
+        (Default ``"most_probable"``) Method to be used for calculating the
+        thermal speed. Valid values are ``"most_probable"``, ``"rms"``,
+        ``"mean_magnitude"``, and ``"nrl"``.
```

Comment:
My only issue with this is that the default value gets lost in the description.  I always get annoyed with documentation that doesn't make default values known at a glance, since I often look to documentation to remind myself.  This is why I tend to list them right away in parenthesis.

I'm up for finding a "better" scheme to standardize this across `plasmapy`.  Maybe creating a Sphinx role like `:default:` that will standardize a formatting.